test/jquery.ime.test.js

```diff
@@ -280,37 +249,6 @@
 		} );
 	} );
 
-	function clusterCaretTest( text, start, end ) {
-		QUnit.test( 'Cursor positioning tests -' + text + '(' + start + ',' + end + ')' , 1, function ( assert ) {
-			var $ced = $( '<div contenteditable="true">' ),
-				correction,
-				position,
-				ime;
-
-			$( '#qunit-fixture' ).append( $ced );
-			$ced.ime();
-			ime = $ced.data( 'ime' );
-
-			$ced.html( text );
-			correction = ime.setCaretPosition( $ced, { start: start, end: end } );
-			position = ime.getCaretPosition( $ced );
-			assert.deepEqual( position, [start - correction[0], end + correction[1] ], 'Caret is at ' + ( start - correction[0] ) + ', ' + ( end + correction[1] ) );
-		} );
-	}
-
-	// @todo FIXME: Unused.
-	clusterCaretTests = [
-		['ക്', 0, 0],
-		['ക്', 1, 1],
-		['ന്ത്', 1, 3],
-		['ന്ത്', 1, 4],
-		['ക്ത്ര', 1, 4]
-	];
-
-	$.each( caretTests, function( i, test ) {
-		clusterCaretTest( test[0], test[1], test[2] );
```

Comment:
Was this Intentionally removed? We have these caret positioning methods, but now part of different classes.